### DISCUSSION TITLE
[A/L] (2021/22) Induzione - Esercizio 20

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#320] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152323349-152e36a8-eca3-4666-ae7d-84b45da8717a.png)

**Comment by Exyss** [comment]
![Esercizio 20](https://user-images.githubusercontent.com/65043207/152661027-4f06c653-82d5-4a40-9a31-d4c9035b9fec.jpg)

**Comment by LuginiAndrea** [comment]
Dimostrazione col principio del minimo numero:
Definiamo la proprietà $P$ come la proprietà espressa nella consegna
Definiamo l'insieme 
$$C = n \geq 1 \wedge P(k) \textrm{ non verificata } \wedge C \neq \emptyset$$
Facile notare come, tolto $0$, il complementare di $C$ siano i valori per i quali $P\left(n\right)$ è verificata
Vediamo se $1 \in C$: 
$$2^1 \cdot 1 \neq (1-1) \cdot 2^2 + 2 \Longleftrightarrow 2 \neq 2 \implies \textrm{ Falso, } 1 \notin C$$
Di conseguenza, $\exists n \notin C \textrm{ t.c. } n = m-1$ (nel caso limite, $n = 1$)

Riscriviamo $m = n + 1$ e controlliamo se $P\left(n+1\right)$ è falsa.
Senza ripetere i calcoli, la versione di @Exyss ci fa vedere come la proprietà è invece verificata, di conseguenza $\nexists m \in \mathbf{N} \textrm{ t.c. } m \in C \Longrightarrow C = \emptyset  \implies \overline{C} = \mathbf{N} - 0$
Quindi la proprietà vale per tutto $\mathbf{N} - 0$, l'insieme sul quale lo volevamo dimostrare
